script/fetch-issues.js

```diff
@@ -223,24 +295,52 @@ function generateRepoInfo(owner, names, issues) {
   return result;
 }
 
+function filterRecent(issues) {
+  // only keep issues created/updated in the past month
+  let recentIssues = [];
```

Comment:
`let` is for a variable that's available to be reassigned, `push` will not reassign the variable, it'll modify the array inside. In this case, keep using `const`.

But see my comment above, you can return the filter return result directly instead without making a variable.